samples/NativeIISSample/NativeIISSample.csproj

```diff
@@ -3,6 +3,9 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Server.IIS\Microsoft.AspNetCore.Server.IIS.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Server.IISIntegration\Microsoft.AspNetCore.Server.IISIntegration.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" />
```

Comment:
Why?

Reply:
Do we still need this seperate sample now that Main is no longer different?

Reply:
Never mind, I meant do we still need a separate test site, but I guess we do because of the web.config?

Reply:
It was for WebHostBuilder. And yes, we need a separate site for the web config ☹️